Like/LikeController.cs

```diff
@@ -0,0 +1,41 @@
+using Blog.Middleware;
```

Comment:
Not how the controller should be, delete it

Reply:
done